docs/designers-developers/developers/tutorials/javascript/loading-javascript.md

```diff
@@ -0,0 +1,42 @@
+# Loading JavaScript
+
+With the plugin in place, you can add the code that loads the JavaScript. This methodology follows the standard WordPress procedure of enqueuing scripts, see [enqueuing section of the Plugin Handbook](https://developer.wordpress.org/plugins/javascript/enqueuing/).
+
+Add the following code to your `myguten-plugin.php` file:
+
+```php
+function myguten_enqueue() {
+	wp_enqueue_script( 'myguten-script',
+		plugins_url( 'myguten.js', __FILE__ )
+	);
+}
+add_action( 'enqueue_block_editor_assets', 'myguten_enqueue' );
+```
+
+The `enqueue_block_editor_assets` hook is used, which is called when the block editor loads, and will enqueue the JavaScript file `myguten.js`.
+
+Create a file called `myguten.js` and add:
+
+```js
+console.log( "I'm loaded!" );
+```
+
+Now start a new post in the block editor, and check your browser Developer Tools. You should see the message in your console log. See [Mozilla's What are browser developer tools?](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_are_browser_developer_tools) if you need more information, the area to become most familiar with is [The JavaScript console](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_are_browser_developer_tools#The_JavaScript_console).
```

Comment:
```suggestion
Next, create a new post in the block editor.

We'll check the JavaScript console in your browser's Developer Tools, to see if the message is displayed. If you're not sure what developer tools are, Mozilla's ["What are browser developer tools?"](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_are_browser_developer_tools) documentation provides more information, including more background on the [JavaScript console](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_are_browser_developer_tools#The_JavaScript_console).

If your code is registered and enqueued correctly, you should see a message in your console:
```

_Then it would be nice to show a screenshot here of the successful console message._